src/components/MessageInput/MessageInput.tsx

```diff
@@ -473,49 +482,20 @@ export const MessageInputWithContext = <
             <Input
               additionalTextInputProps={additionalTextInputContainerProps}
               getUsers={getUsers}
-              handleOnPress={handleOnPress}
+              handleOnPress={toggleAttachmentPicker}
```

Comment:
makes sense to change the name of the prop on `Input` to `toggleAttachmentPicker` instead of `handleOnPress` now

Reply:
Done!!